### PR TITLE
Test extract metadata

### PR DESCRIPTION
**Pull request by jpgonzales27**

### CODE REVIEW

File: src/test/java/org/fundacion/jala/converter/core/ExtractMetadataTest.java [hunk 1]
```diff
@@ -0,0 +1,35 @@
+package org.fundacion.jala.converter.core;
+
+import org.fundacion.jala.converter.core.exceptions.MetadataException;
+import org.fundacion.jala.converter.core.metadata.TypeFileExport;
+import org.junit.Test;
+
```
Comment (JorgeRCV) [inline src/test/java/org/fundacion/jala/converter/core/ExtractMetadataTest.java]: Delete tis unnecessary space

File: src/test/java/org/fundacion/jala/converter/core/ExtractMetadataTest.java [hunk 2]
```diff
@@ -0,0 +1,35 @@
+package org.fundacion.jala.converter.core;
```
Comment (JorgeRCV) [inline src/test/java/org/fundacion/jala/converter/core/ExtractMetadataTest.java:1]: Copyright doc is missing in all classes

File: src/test/java/org/fundacion/jala/converter/core/ExtractMetadataTest.java [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+
+import org.fundacion.jala.converter.core.exceptions.MetadataException;
+import org.fundacion.jala.converter.core.metadata.TypeFileExport;
+import org.junit.Test;
+
+import java.io.File;
+
+public class ExtractMetadataTest {
+
```
Comment (cristianfchq) [inline src/test/java/org/fundacion/jala/converter/core/ExtractMetadataTest.java:9]: I think that is not necesary this line.

This line should be removed

File: src/test/java/org/fundacion/jala/converter/core/ObjectMetadataTest.java [hunk 4]
```diff
@@ -0,0 +1,65 @@
+package org.fundacion.jala.converter.core;
```
Comment (cristianfchq) [inline src/test/java/org/fundacion/jala/converter/core/ObjectMetadataTest.java:1]: Is missing Copyright for this file

File: src/test/java/org/fundacion/jala/converter/core/metadata/ExportTypeFileTest.java [hunk 5]
```diff
@@ -0,0 +1,30 @@
+package org.fundacion.jala.converter.core.metadata;
```
Comment (cristianfchq) [inline src/test/java/org/fundacion/jala/converter/core/metadata/ExportTypeFileTest.java:1]: Is missing Copyright for this file

Comment (jpgonzales27) [reply src/test/java/org/fundacion/jala/converter/core/metadata/ExportTypeFileTest.java:1]: We agreed that javadoc and copyright would not be put in the tests

File: src/test/java/org/fundacion/jala/converter/core/ExtractMetadataTest.java (hunk 2 above)
Comment (jpgonzales27) [reply src/test/java/org/fundacion/jala/converter/core/ExtractMetadataTest.java:1]: We agreed that javadoc and copyright would not be put in the tests

File: src/test/java/org/fundacion/jala/converter/core/ObjectMetadataTest.java (hunk 4 above)
Comment (jpgonzales27) [reply src/test/java/org/fundacion/jala/converter/core/ObjectMetadataTest.java:1]: it's already corrected cris

File: src/test/java/org/fundacion/jala/converter/core/ExtractMetadataTest.java (hunk 3 above)
Comment (jpgonzales27) [reply src/test/java/org/fundacion/jala/converter/core/ExtractMetadataTest.java:9]: it's already corrected cris

File: src/test/java/org/fundacion/jala/converter/core/ExtractMetadataTest.java (hunk 1 above)
Comment (jpgonzales27) [reply src/test/java/org/fundacion/jala/converter/core/ExtractMetadataTest.java]: it's already corrected cris

**Review by JorgeRCV — APPROVED** [review]

**Review by cristianfchq — APPROVED** [review]

**Pull request merged**